src/lib/serviceRoutinesV2/getEntities.cpp

```diff
@@ -314,31 +314,23 @@ std::string getEntities
 
   // 03. Render Entities response
 
-  if (parseDataP->qcrs.res.contextElementResponseVector.size() == 0)
```

Comment:
Not sure if the proposed change is correct...

It seems you have removed some checks (the one for `parseDataP->qcrs.res.contextElementResponseVector.size()` in the original external if clause and the one for `oe.code != SccNone` in the original internal if clause), I understand that with the aim of simpliying the logic.

However, although I don't remember why, I understand these checks are here for some reason :). In fact, if you look to the travis run of the functional tests at https://travis-ci.org/telefonicaid/fiware-orion/jobs/643774476?utm_medium=notification&utm_source=github_status you will see that some of the them, probably for this reason (did you run the test locally before creating the PR?)

Not sure, but maybe the issue should be solved in a lower part (in mongoBackend)